### PR TITLE
Activate hamburger

### PR DESCRIPTION
**Pull request by Oscah01**
-add config files
-activated hamburger menu

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -168,8 +124,8 @@ header > .logo {
   background-size: 100%;
   background-repeat: no-repeat;
   border-radius: 0 0 0 90px;
-  padding: 100px 20px 0 20;
   background-color: white;
+  filter: blur(5px);
```
Comment (ShahierNashaat) [inline style.css:128]: - [ ] Please remove the filter as it makes the header not readable.

Screenshot from 2022-07-26 19-35-10

File: index.html [hunk 2]
```diff
@@ -28,12 +28,30 @@
...
-          <span class="bar"></span>
-        </div>
       </nav>
+        <div class="container12"></div>
+        <div class="navbar">
+          <div class="hamburger-menu">
+            <div class="line  line-1"></div>
+            <div class="line line-2"></div>
+            <div class="line line-3"></div>  
+            </div>
```
Comment (ShahierNashaat) [inline index.html:38]: - [ ] Kindly be noticed that when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

File: style.css [hunk 3]
```diff
@@ -506,7 +462,90 @@ header > .logo {
...
+
+.change {
+  right: 0;
+}
+
+.hamburger-menu {
+  width: 35px;
+  height: 30px;
+  position: fixed;
+  top: 50px;
```
Comment (ShahierNashaat) [inline style.css:488]: - [ ] Please make the top: 20px as it should be aligned with the logo.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Oscah01  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**